src/label_studio_sdk/projects/client_ext.py

```diff
@@ -28,8 +28,14 @@ def list(self, **kwargs) -> SyncPagerExt[T]:
     list.__doc__ = ProjectsClient.list.__doc__
 
     def get(self, id: int, *, request_options: typing.Optional[RequestOptions] = None) -> ProjectExt:
-        return ProjectExt(**dict(super().get(id, request_options=request_options)))
-    
+        return typing.cast(
+            ProjectExt,
+            construct_type(
+                type_=ProjectExt,  # type: ignore
+                object_=super().get(id, request_options=request_options).model_dump(),
+            ),
+        )
```

Comment:
this ended up being kind of interesting - the original implementation triggered Pydantic validation, which following Fern's implementation, we don't actually want.